aws/resource_aws_backup_plan_test.go

```diff
@@ -31,11 +32,13 @@ func TestAccAwsBackupPlan_basic(t *testing.T) {
 					testAccMatchResourceAttrRegionalARN(resourceName, "arn", "backup", regexp.MustCompile(`backup-plan:.+`)),
 					resource.TestCheckResourceAttr(resourceName, "name", rName),
 					resource.TestCheckResourceAttr(resourceName, "rule.#", "1"),
-					testAccCheckAwsBackupPlanRuleAttr(resourceName, &ruleNameMap, rName, "rule_name", rName),
```

Comment:
While we're dropping this custom `TestCheckFunc` I think it'd also be worth getting rid of the `ruleNameMap` handling completely in all the test functions and `testAccCheckAwsBackupPlanExists()`. 👍

Reply:
agreed! removed in b6b9297